packages/utilities-deprecated/projects/utilities/src/js/index.ts

```diff
@@ -10,7 +10,7 @@ declare global {
 }
 
 const isCdnCn = typeof window !== 'undefined' && window.PORSCHE_DESIGN_SYSTEM_CDN === 'cn';
-export const FONT_FACE_CDN_URL = (isCdnCn ? 'https://cdn.ui.porsche.cn' : 'https://cdn.ui.porsche.com') + '/porsche-design-system/styles/' + (isCdnCn ? 'font-face.min.cn.ec02d2159cb7c560e10be175ce8af166.css' : 'font-face.min.13dbdc94bb389e67a5c1bd140e500366.css');
+export const FONT_FACE_CDN_URL = (isCdnCn ? 'https://cdn.ui.porsche.cn' : 'https://cdn.ui.porsche.com') + '/porsche-design-system/styles/' + (isCdnCn ? 'font-face.min.cn.dbb9b1f73aba20c5fad38212405e1a68.css' : 'font-face.min.955063f599f3c828248b49f258753ba8.css');
```

Comment:
not 100% sure, if this is really good. since old utilities package still supports thin font-weight, if not than it would be fine.

Reply:
we'll keep it for now, but we need to check if it's good after re-activating the vrt tests for deprecated utilities package